### DISCUSSION TITLE
Brazil, the talent slayer

### DISCUSSION THREAD

**Discussion by Reskalla** [community/community#141941] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

This is a bit of a unique topic. It’s more like a testimony, fueled by my experiences with the world. I want to record this story somewhere, the story of a Brazilian scientist who was completely overshadowed by the historical culture of “Brazilian factory workers.”

If you’re born in Brazil and you’re “outside the box,” not caring about gyrating butts on TV, parties where people rub against each other in dark corners with loud music, or a bunch of grown-ups chasing a ball without knowing the geometry of water or how many planets are in the solar system, you tend to be overshadowed by a society that spends rivers of money on trivial, banal things that don’t bring a future to the planet or society. If you’re not exceptionally good and completely devoid of any patriotic feelings to the point of moving out of the country and trying your luck elsewhere, you’ll be erased and overshadowed in this country. It’s no surprise that I’m such a case.

I have numerous projects that would be revolutionary for the planet, like water heating by electromagnetism (different from industrial methods, using 10 times less energy), molecular cooling by electromagnetism (revolutionizing refrigerators), energy storage by photon packets (eliminating our need to use so much chemistry to make batteries, using physical principles instead), wireless energy transmission through particular waves (making straight trajectories from curve to curve with quantum principles), modeling carbon dioxide into water and alcohol (by bombarding photons with microwave frequencies and ionized hydrogen), etc. I really have countless projects, but when, with great effort, they get off the ground, something incredibly convenient happens in the world to overshadow my creations.

I have a sharp mind, quick logical thinking, I learn everything on my own, the ability to see anything in 3D in my mind, to handle, disassemble, see every screw… But it’s hard to stay convinced on the path, with so many “no’s” and doors being closed by fate.
I made a prototype of an electric shower with electromagnetic heating, using 96W, with a 12V 8Ah battery connected to a mini solar panel. I made a gate controller with fingerprint recognition and RFID transfer that, upon identifying an unregistered fingerprint, would ring the doorbell or call the owner’s phone via video. I used it at home for 8 months until I lost the company I built with so much effort to Covid. I designed molecular cooling for metals by molecular braking, transporting controlled rhythm, but I never managed to build the prototype due to lack of food at home. Etc… Etc… Etc…
At 12 years old, I knew how to assemble a single-engine airplane motor from World War II. I was always on the winning team in school science projects. But since I was always anti-social, preferring to stay home playing video games rather than going to parties with schoolmates, I was never given an opportunity. After all, who would invest money in a poor child, without society’s attention, who doesn’t kick a ball on a grassy field, doesn’t kiss 12 girls at a party, and has no political contacts? How I wish I had been born in a serious country, I guarantee this world would be much more advanced by now.

Not only did I want to leave, but I also tried several times. I started learning and speaking Japanese because I wanted to move to Japan. But being a closed country, I needed to demonstrate my skills and have a good amount of money in the bank. The first was easy, but the second, although not impossible, became so with the birth of my son (the most wonderful thing fate gave me). A post-pandemic child, a company struggling to stay afloat with closed commerce, my CNC machine broke during the move, the 3D printer was sold to pay bills… All the money I saved was lost, and now with an added difficulty for a poor man (diapers, medicine, medical care, new clothes, diapers, wipes, milk, vitamins, new clothes, diapers, protein, little shoes, new clothes again, potty, etc., etc., diapers, etc.).

I had to abandon my childhood dream of living in Japan, my country at heart. I started studying Finnish until I encountered new problems in Brazil… Unemployment, hunger, computer broke, phone broke, lack of interest in technology, dangerous neighborhoods… I lost count of how many times I ate plain bread with water.
I recently tried to get government aid for a donated computer, basic food basket, etc… But since it’s election season, they can’t help the population to avoid appearing as vote-buying… Do you think the lack of food on my table cares about vote-buying?
Without a computer, I can’t even build a prototype to submit a portfolio on Kickstarter.

Anyway. I hope this testimony recorded here lets people know that, as the Brazilian song says:
“The cure for AIDS died of hunger in the interior of Maranhão… The cure for cancer died of schistosomiasis in the interior of Minas Gerais… Physicists, musicians, mathematicians… Being saved from hunger by crabs in the mangrove. (…) The Brazilian Nobel Prize winner… Sniffs glue and begs for alms in downtown Rio de Janeiro.”

Thank you for reading this far and sorry for the sincere testimony. I’m just tired of seeing so many sharp and creative minds starving in this country.

**Comment by Berchez** [comment]
Your story is moving and reflects the difficulties that many creative Brazilians face. Your ideas are incredible and demonstrate your talent, even in the face of so many barriers. I understand your frustration, but don’t give up. There are global networks and platforms, such as Kickstarter, that can help give visibility to your projects. Your value goes beyond what the current system recognizes. Keep believing in your creations — they have the potential to make a difference. Btw, I'm also a brazilian.
